src/main/java/uk/gov/ida/matchingserviceadapter/MatchingServiceAdapterConfiguration.java

```diff
@@ -140,6 +144,10 @@ public EuropeanIdentityConfiguration getEuropeanIdentity() {
         return europeanIdentity;
     }
 
+    public boolean shouldSignWithSHA1() {
```

Comment:
this is me just crying that Java still hasn't figured out how to get rid of getters and setters 😭 No action needed